lib/gist.rb

```diff
@@ -110,10 +111,15 @@ def multi_gist(files, options={})
     json[:files] = {}
 
     files.each_pair do |(name, content)|
-      raise "Cannot gist empty files" if content.to_s.strip == ""
-      json[:files][File.basename(name)] = {:content => content}
+      if content.to_s.strip == ""
+        raise "Cannot gist empty files" unless options[:skip_empty]
+      else
+        json[:files][File.basename(name)] = {:content => content}
+      end
     end
 
+    raise "All files were empty. No gist created." if json[:files].empty? && options[:skip_empty]
```

Comment:
@phallstrom I think the program should not crash if you pass `--skip-empty` and an empty file. It should just silently exit successfully.

Reply:
Ah, I misread the original ticket thinking you _wanted_ it to output details. Patch up.